CHANGES

```diff
@@ -1,5 +1,47 @@
 Revision history for Shaderc
 
+v2020.0 2020-01-17
```

Comment:
If this is the version that is still in progress, this this should have "-dev" in it.